documents/openapi/openapi.yaml

```diff
@@ -1,32 +1,15 @@
 openapi: 3.0.0
 info:
   title: VA Notify API Documentation
-  description: "This documents the API schemas for consumption by internal VA developers.<br />
-  <h1>Authorization header</h1>
-  <p>You must include an authorization header that is encoded using JSON Web Tokens.
-  JSON Web Tokens have a standard header and a payload. The header consists of:
-  <pre><code>
-  {\n
-  &nbsp;&nbsp;&nbsp;&nbsp;&quot;type&quot;: &quot;JWT&quot;,\n
-  &nbsp;&nbsp;&nbsp;&nbsp;&quot;alg&quot;: &quot;HS256&quot;\n
-  &nbsp;}
-  </code></pre>
-  </p>
-  <p>
-  The payload consists of `iss` and `iat`. `iss` is the `service_id` that is sending the notification, and `iat` is
-  the 'Issued At' timestamp in seconds since Unix epoch. Here is an example payload:
-  <pre><code>
-  {\n
-  &nbsp;&nbsp;&nbsp;&nbsp;&quot;iss&quot;: &quot;26785a09-ab16-4eb0-8407-a37497a57506&quot;,\n
-  &nbsp;&nbsp;&nbsp;&nbsp;&quot;iat&quot;: 1568818578\n
-  &nbsp;}
-  </code></pre>
-  </p>
-  <p>
-    Refer to the <a href='https://jwt.io/' target='_blank'>JSON Web Tokens website</a> for more information
-    on encoding your authorization header.
-  </p>"
-  version: 1.0.0
+  version: 1.0.1
+  description: |
+    "<p>This documents the API schemas for consumption by internal VA developers.</p>
+    <h1>Authorization header</h1>
+    <p>You must include an authorization header that is encoded using JSON Web Tokens.  JSON Web Tokens have a standard header and a payload. The header consists of:</p>
+    <code>{ &quot;type&quot;: &quot;JWT&quot;, &quot;alg&quot;: &quot;HS256&quot; }</code>
+    <p>The payload consists of `iss` and `iat`. `iss` is the `service_id` that is sending the notification, and `iat` is the 'Issued At' timestamp in seconds since Unix epoch. Here is an example payload:</p>
+    <code>{ &quot;iss&quot;: &quot;26785a09-ab16-4eb0-8407-a37497a57506&quot;, &quot;iat&quot;: 1568818578 }</code>
+    <p>Refer to the <a href='https://jwt.io/' target='_blank'>JSON Web Tokens website</a> for more information on encoding your authorization header.</p>"
```

Comment:
I preferred the original formatting because in the UI it looks cleaner and changing this isn't part of the ticket.

Reply:
I used the Swagger editor to restore the original display whilst still cleaning up the markup.